ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/RawQueryStatusCount.java

```diff
@@ -0,0 +1,70 @@
+package io.confluent.ksql.rest.entity;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.google.common.base.Joiner;
+import java.util.Collections;
+import java.util.EnumMap;
+import java.util.Map;
+import java.util.Objects;
+import org.apache.kafka.streams.KafkaStreams;
+import org.apache.kafka.streams.KafkaStreams.State;
+
+/**
+ * Used to keep track of a the state of KafkaStreams application
+ * across multiple servers. Used in {@link RunningQuery}.
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class RawQueryStatusCount {
```

Comment:
as discussed offline, let's move this to a different PR (and we can keep the while loop checking for non-errors for now) as this affects production code

Reply:
Done